Makefile

```
@@ -0,0 +1,8 @@
+default: style-check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)